### PR TITLE
feat: update to hydra-genetics version 0.9.2

### PR DESCRIPTION
**Pull request by Smeds**
### Review and tests: 
- [ ] Tests pass

### CODE REVIEW

File: workflow/rules/common.smk [hunk 1]
```diff
@@ -64,7 +68,7 @@ def generate_read_group(wildcards):
         "{}.{}".format(wildcards.sample, wildcards.lane),
```
Comment (jonca79) [inline workflow/rules/common.smk:68]: "{}_{}.{}.{}".format(unit.sample, unit.type, unit.lane, unit.barcode),

File: workflow/rules/common.smk [hunk 2]
```diff
@@ -64,7 +68,7 @@ def generate_read_group(wildcards):
...
         "{}_{}".format(wildcards.sample, wildcards.type),
         get_unit_platform(units, wildcards),
-        "{}.{}.{}".format(get_unit_flowcell(units, wildcards), wildcards.lane, get_unit_barcode(units, wildcards)),
+        "{}.{}.{}".format(get_unit_flowcell(units, wildcards), wildcards.lane, get_unit_barcodes(units, wildcards)),
     )
```
Comment (jonca79) [inline workflow/rules/common.smk:72]: In the parabricks module the LB flag is also set:
"{}_{}".format(unit.sample, unit.type),

Comment (Smeds) [reply workflow/rules/common.smk:73]: added

File: workflow/rules/common.smk (hunk 1 above)
Comment (Smeds) [reply workflow/rules/common.smk]: updated

**Review by jonca79 — APPROVED** [review]

**Pull request merged**